marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubConfigImpl.java

```diff
@@ -1947,8 +1947,6 @@ private void updateAppConfig(AppConfig config) {
 
         initializeModulePaths(config);
 
-        config.setSchemasPath(getUserSchemasDir().toString());
```

Comment:
setSchemasPath was deprecated in ml-app-deployer 3.x and removed in 4.0 - this line of code was essentially having no impact, as the "user schemas dir" is src/main/ml-schemas, which is the default for ml-app-deployer (though it's better to use ml-config/databases/(db name)/schemas).

Reply:
We might want to make sure we note the backwards dependency, as folks who were using the user schemas directory under older versions, will suddenly find their configs will not work when they upgrade.

Reply:
Looking at HubProjectImpl, getUserSchemasDir is set to src/main/ml-modules, and a user can't override that. That's also the default in ml-app-deployer. So that's why I say that line of code was having no impact.